src/bussinessLogic/ManageMap.java

```diff
@@ -56,38 +43,40 @@ public void setNewCharMap() {
 
         for (int i = 0; i < Constants.NUMBER_OF_LETTERS; i++) {
             charString = Character.toString(currentChar);
-            // charMap.put(charString, 0);
             map.put(charString, new LetterAttributes(0, 0));
-            dividerMap.put(charString, 0);
             currentChar++;
         }
     }
 
     /**
-     * sort the charMap in ascending order
+     * transfers the chars from map to rankingArray and
+     * arranges each char relative to its correlated reaction time
      */
-    public void sortCharMap() {
-        // generate a new, empty sortedMap
-        sortedMap = new LinkedHashMap<>();
-        // arrayList for storing and sorting the chars
+    public void rankChars() {
+
```

Comment:
This line can be deleted.

Reply:
Yes.

In the future, to format the code accordingly, you the formatting features of Eclipse or IntelliJ. They are quite useful.